test/cookbooks/osl-git-test/metadata.rb

```diff
@@ -10,3 +10,4 @@
 depends          'osl-git'
 
 supports         'centos', '~> 7.0'
+supports         'centos', '~> 8.0'
```

Comment:
There's no need for this in this metadata.rb. Please go ahead and remove all of these.